wpstarter/templates/wp-config.example

```diff
@@ -16,6 +16,11 @@
 require_once realpath({{{VENDOR_PATH}}}.'/autoload.php');
 
 /**
+ * A reference to `.env` folder path.
+ */
+define('WPSTARTER_PATH', {{{ENV_REL_PATH}}});
```

Comment:
@dnaber-de All the variables used inide the `wp-config.example` template are generated during installation in the wp-config "step".

See https://github.com/wecodemore/wpstarter/blob/master/wpstarter/src/Setup/Steps/WPConfigStep.php#L99-L111